File: in_Java/0167 Two Sum II - Input array is sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i = 0;
        int j = numbers.length-1;
        int t;
        int[] res = new int[2];
        while(i < j){
            t = numbers[i] + numbers[j];
            if(t == target){
                res[0] = i+1;
                res[1] = j+1;
                break;
            }else if(t < target){
                i++;
            }else{
                j--;
            }
        }
        return res;
    }
}

File: in_Python/0167 Two Sum II - Input array is sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        i = 0
        j = len(numbers) - 1
        res = []
        while i < j:
            if numbers[i] + numbers[j] == target:
                res.append(i+1)
                res.append(j+1)
                return res
            elif numbers[i] + numbers[j] < target:
                i += 1
            else:
                j -= 1
        return res